src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/dataset_retriever.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
More of a general comment but the directory is more about inputs now and not LLM specific. Can we update the directory name to inputs?

Reply:
Thats the right call. Appreciate it.